<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoActions</class>
 <widget class="QWidget" name="AutoActions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>45</number>
   </property>
   <property name="bottomMargin">
    <number>45</number>
   </property>
   <item row="0" column="0">
    <widget class="QLineEdit" name="link"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="setLink">
     <property name="text">
      <string>Вставить ссылку</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="start">
     <property name="text">
      <string>Запустить все</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Выбрать у всех</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="like">
         <property name="text">
          <string>Лайк</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="comm">
         <property name="text">
          <string>Коментарий</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
